rst_files/getting_started.rst

```diff
@@ -103,21 +50,20 @@ Installing Julia and Dependencies
 
 .. While using the Docker instance is convenient and error-proof, you may eventually want to install things locally
 
-The easiest approach is to using Julia with Jupyter on your desktop is to install Anaconda
+The easiest approach is to using Julia with Jupyter on your desktop is to install Anaconda and then Julia
 
 1. Install Anaconda by `downloading the binary <https://www.anaconda.com/download/>`_ (3.7 version)
 
-    * Make sure you click yes to "add Anaconda to my PATH."
-    * If you would rather do that later, 
-        see: `Anaconda for Windows <http://docs.anaconda.com/anaconda/install/windows/>`_ and
-        `Mac/Linux <https://conda.io/docs/user-guide/install/macos.html>`_.
+    * Make sure you click yes to "add Anaconda to my PATH"
+
+2. Download and install Julia, from `download page <http://julialang.org/downloads/>`_ , accepting all default options
 
-2. Download and install Julia, from `download page <http://julialang.org/downloads/>`_ , accepting all default options.
+    * Do not install JuliaPro due to its limited number of available packages
 
 3. Open Julia, either by
 
     * Navigating to Julia through your menus or desktop icons (Windows, OSX), or
-    * Opening a terminal and typing ``julia`` (Linux, + OSX/git bash if you configure it)
+    * Opening a terminal and typing ``julia`` (Linux, OSX, Windows)
```

Comment:
@jlperla Last I checked, this was not automatic on OSX/Windows (i.e., installing gets you the binary, but it's up to you to plug into your terminal). And that's what it says [here](https://en.wikibooks.org/wiki/Introducing_Julia/Getting_started#Running_directly_from_terminal), which is written for Julia v1.0.0

Reply:
Removed per Slack. Less options are better.